src/NuGet.Core/NuGet.Common/PublicAPI.Unshipped.txt

```diff
@@ -1 +1,6 @@
 #nullable enable
+NuGet.Common.PublicStrings
+~static NuGet.Common.PublicStrings.Culture.get -> System.Globalization.CultureInfo
+~static NuGet.Common.PublicStrings.Culture.set -> void
+~static NuGet.Common.PublicStrings.Invalid_AttributeValue.get -> string
```

Comment:
this is just my personal opinion, but I don't like resource strings as public APIs. There's too much risk of runtime failures if new `{x}` parameters are added to the string, and callers of the API don't update their `string.Format` argument list.  We've even had this type of failure within the repo, so it'll be much worse if anyone using the NuGet Client SDK packages in their own app use this string for some reason.

I don't think that duplicating the strings, and keeping them internal, in multiple assemblies is bad enough to prefer public APIs.

Reply:
I want 6, but let's wait on that until we have a major version. 

I'd prefer 1

Reply:
Okay I've pushed an update to use a shared resource that gets compiled into each assembly that needs it.  At least the string is only defined in one place.